controllers/clients.js

```diff
@@ -1,136 +1,80 @@
-const fs = require('fs');
-const clients = require('../data/clients.json');
+const Clients = require('../models/Clients');
 
-const errorResHelper = (errorDescription, res, errCode = 400) => {
-  res.status(errCode).json({ error: errorDescription });
+const listClients = (req, res) => {
+  Clients.find()
+    .then((clients) => res.status(200).json(clients))
+    .catch((error) => res.status(400).json(error));
 };
 
 const createClient = (req, res) => {
-  const now2ISO = new Date().toISOString();
-  const missing = [];
-  const newClient = {
-    id: new Date().getTime().toString(),
-    name: req.query.name ?? missing.push("'name'"),
-    phone: req.query.phone ?? missing.push("'phone'"),
+  const client = new Clients({
+    name: req.body.name,
+    phone: req.body.phone,
     location: {
-      country: req.query.country ?? missing.push("'country'"),
-      state: req.query.state ?? missing.push("'state'"),
-      city: req.query.city ?? missing.push("'city'"),
-      address: req.query.address ?? missing.push("'address'"),
+      country: req.body.location.country,
```

Comment:
do:
```
location: req.body.location
```

instead of: 
```
location: {
      country: req.body.location.country,
      state: req.body.location.state,
      city: req.body.location.city,
      address: req.body.location.address,
    },
```

Reply:
Chaged it, great tip!